src/06-time-tracking/time-tracking.apib

```diff
@@ -100,12 +100,6 @@ Add tracked time.
     + Attributes (object)
         + work_type_id: `2175597d-484e-4a1c-a781-cbc3d9f893ba` (string, optional)
         + One Of
-            + Properties
```

Comment:
This makes no sense, right? let's remove this level and require started_at & duration for now

Reply:
agreed to do this if needed, or add the ended_at soon so this is no longer an issue :-)